frontend/src/components/TaskView/FutureView/FutureViewDay.tsx

```diff
@@ -162,3 +183,10 @@ export default function FutureViewDay(props: Props): ReactElement {
     </div>
   );
 }
+
+const Connected = connect(
+  ({ settings: { theme } }: State, ownProps: Props): Props & { readonly theme: Theme } => (
+    { ...ownProps, theme }
```

Comment:
Correct me if I'm wrong, but the logic here seems unnecessary. If you do
```tsx
const Connected = connect(
  ({ settings: { theme } }: State):  { readonly theme: Theme } => ({  theme }),
)(FutureViewDay);
```
it's equivalent to what you did here. `react-redux` automatically handles this common case.